web/pgadmin/browser/server_groups/servers/databases/publications/__init__.py

```diff
@@ -381,24 +394,63 @@ def _fetch_properties(self, did, pbid):
             return False, gone(self._NOT_FOUND_PUB_INFORMATION)
 
         if not res['rows'][0]['all_table']:
-            get_name_sql = render_template(
-                "/".join([self.template_path, self._DELETE_SQL]),
-                pbid=pbid, conn=self.conn
-            )
-            status, pname = self.conn.execute_scalar(get_name_sql)
-            table_sql = render_template(
-                "/".join([self.template_path,
-                          self._GET_TABLE_FOR_PUBLICATION]),
-                pname=pname
-            )
+            if self.manager.version < 150000:
+                get_name_sql = render_template(
+                    "/".join([self.template_path, self._DELETE_SQL]),
+                    pbid=pbid, conn=self.conn
+                )
+                status, pname = self.conn.execute_scalar(get_name_sql)
+                table_sql = render_template(
+                    "/".join([self.template_path,
+                              self._GET_TABLE_FOR_PUBLICATION]),
+                    pname=pname
+                )
 
-            pub_table = []
-            status, table_res = self.conn.execute_dict(table_sql)
+                pub_table = []
+                pub_table_names_list = []
+                status, table_res = self.conn.execute_dict(table_sql)
 
-            for table in table_res['rows']:
-                pub_table.append(table['pubtable'])
+                for table in table_res['rows']:
+                    pub_table.append(table['pubtable'])
+                    pub_table_names_list.append(table['pubtable'])
+
+            elif self.manager.version >= 150000:
+                schema_name_sql = render_template(
+                    "/".join([self.template_path, 'get_schemas.sql']),
+                    pbid=pbid
+                )
+                status, snames_list_res = self.conn.execute_dict(
+                    schema_name_sql)
+
+                snames_list = []
+
+                if len(snames_list_res['rows']) != 0:
+                    for sname_dict in snames_list_res['rows']:
+                        snames_list.append(sname_dict['sname'])
+
+                if len(snames_list) != 0:
+                    res['rows'][0]['pubschema'] = snames_list
```

Comment:
just use a list comprehension here to assign `sname` array directly. remove all over if codes.